### PR TITLE
Replace nose with py.test

### PR DESCRIPTION
**Pull request by jgmize**

### CODE REVIEW

**Comment by jgmize** [comment]
This will require a change to the Jenkins job config before it will pass.

**Comment by jgmize** [comment]
To run tests locally:
fig build web
fig run web bin/unit_tests

**Comment by jgmize** [comment]
I also added a script that will run arbitrary commands in a running container, so if you've already run `fig up` you can run `bin/execweb bin/unit_tests` and it will take even less time.

I'm open to different names for the scripts; the intention is to use py.test markers to differentiate true unit tests from integration tests that use phantomjs and/or selenium/sauce labs.

**Comment by jgmize** [comment]
@glogiotatidis r?

**Comment by glogiotatidis** [comment]
I like the grouping tests in classes so that a function or class has all it's related tests in a test class. I believe it keeps code more tidy. I don't have strong feelings about this I'm just stating my preference :)

**Comment by glogiotatidis** [comment]
You can rebase this and modify bin/jenkins/runtests.sh to run the new test commands.

File: bin/unit_tests [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+export DJANGO_SETTINGS_MODULE=masterfirefoxos.settings
+py.test
```
Comment (glogiotatidis) [inline bin/unit_tests]: Change this to `py.test masterfirefoxos` to limit masterfirefoxos directory. Helps if we have venv directories around.

File: bin/unit_test_coverage [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+export DJANGO_SETTINGS_MODULE=masterfirefoxos.settings
+py.test --cov masterfirefoxos --cov-report=term-missing
```
Comment (glogiotatidis) [inline bin/unit_test_coverage]: Same here, add `masterfirefoxos` at the end `py.test --cov masterfirefoxos --cov-report=term-missing masterfirefoxos`

**Comment by glogiotatidis** [comment]
a few nits only. works great, i like it! r+ :octopus:

**Comment by jgmize** [comment]
Thanks @glogiotatidis! I rebased and applied your suggested changes (except for re-grouping everything back into classes; we'll have to agree to disagree on that one ;) and Jenkins is happy now.

**Pull request merged**